WEB_SERVICE/WEB/_HARDENING/WEB_HTTP_CSP/description.md

```diff
@@ -1,57 +1,47 @@
-Content Security Policy (CSP) is a computer security standard that provides an added layer of protection against
-Cross-Site Scripting (XSS), clickjacking, and other client-side attacks that rely on executing malicious content in the
-context of a trusted web resource.
+**Content Security Policy** (CSP): is a computer security standard that provides an added layer of protection against Cross-Site Scripting (XSS), clickjacking, and other client-side attacks that rely on executing malicious content in the context of a trusted web resource.
```

Comment:
```suggestion
**Content Security Policy** (CSP): A computer security standard that provides an added layer of protection against Cross-Site Scripting (XSS), clickjacking, and other client-side attacks that rely on executing malicious content in the context of a trusted web resource.
```

Reply:
@Zakaria-Latif, I think this is out of the PR context.

Reply:
The PR says `fix display`, I thought it was meant to correct typos, grammar, and spelling !!.